pkg/mount/nfs.go

```diff
@@ -67,6 +67,11 @@ func (m *nfsMounter) Reload(source string) error {
 
 	err = m.reload(source, newNFSmounter.mounts[source])
 	m.LogDevices()
+
+	if m.mounts[source] == nil {
```

Comment:
This will end up logging every time Reload() is called when source is not mounted which might be a genuine case.
Instead before "reload()" is called check if m.mounts[source] exists and if it exists and now it is nil then log them.